RELEASE.md

```diff
@@ -1,3 +1,45 @@
+# Release 1.12.0
+
+## Major Features and Improvements
+* Keras models can now be directly exported to the SavedModel format(`tf.contrib.saved_model.save_keras_model()`) and used with Tensorflow Serving.
+* Keras model now supports evaluating with a `tf.data.Dataset`.
+* TensorFlow binaries are built with XLA support linked in by default.
+
+## Bug Fixes and Other Changes
+
+* tf.data:
+  * tf.data users can now represent, get, and set options of TensorFlow input pipelines using `tf.data.Options()`, `tf.data.Dataset.options()`, and `tf.data.Dataset.with_options()` respectively.
+* tf.data users can now use `tf.data.experimental.AUTOTUNE` for the num_parallel_calls argument of `tf.data.Dataset.interleave`, `tf.data.Dataset.map`, and `tf.data.experimental.map_and_batch` to dynamically tune the level of parallelism at runtime.
```

Comment:
This worked very poorly for me and I believe there is still work being done.  I do not know if I would mention it just yet.  I did some more tests and found I wasn't actually using the AUTOTUNE and when I did things go worse than when I set the values myself.  This may not be everyone's experience but it was with official/resnet.

Reply:
SGTM

Reply:
Done.